enrollment-server/src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,8 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/${JDBC_DATABASE_SCHEMA}
+spring.datasource.username=${JDBC_DATABASE_USERNAME}
+spring.datasource.password=${JDBC_DATABASE_PASSWORD}
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
```

Comment:
PostgreSQLDialect is deprecated. We may want to start using specific versions. Feel free to raise this as a separate issue.